src/Repositories/SearchIndexRepository.php

```diff
@@ -217,56 +214,43 @@ protected function getPrefixedTerms($term)
      *
      * @return void
      */
-    public function createSearchIndexes($brand)
+    public function createSearchIndexes(): void
```

Comment:
When touching large functions like this i'd love to see some effort put into making them more readable and this is a perfect example.  Also the name create does not fully encompass what this function does.  RebuildSearchIndexes is probably more descriptive.

My ideal would be something like this:
```
function rebuildSearchIndexes(){
   $lastUpdate = getTimeLastUpdateCompleted();
   
   $this->updatePostSearchIndexes(...);
   $this->updateThreadSearchIndexes(...);
}
```

Reply:
Good point. Thanks for the suggestion!